app/index.js

```diff
@@ -1,13 +1,30 @@
-var app = require('./app');
-var AppConfig = require('./load_config');
-var restartJobModule=require('./restart-jobs')
-var logger=require('./logger')
+const AppConfig = require('./load_config');
+const restartJobModule = require('./restart-jobs');
+const logger = require('./logger');
+const grpc = require('./grpc');
+const http = require('./http');
+const discovery = require('./discovery');
+
+let httpPort = AppConfig.httpPort;
+let grpcPort = AppConfig.grpcPort;
+let sslConfig = AppConfig.ssl;
+let mode = AppConfig.mode;
+
+if (mode === 'grpc') {
+  grpc.start(grpcPort, sslConfig);
+}
+
+http.start(httpPort);
```

Comment:
Should we consider an option to disable http protocol, because Eureka supports one port per instance that means either we register grpc port or http port.

Reply:
currently eureka registers only grpc port, so does not seem an issue to me

Reply:
also after migration to grpc is finished, there will be no http protocol.